Done parts of Simplified Chinese translations.

heerxingen opened the pull request:
Hi I saw your mod on steam and I quite like it, so I made some translations. It's not completed nor perfect but I plan to fix that in the future. Nice mod!

heerxingen committed:
Done parts of Simplified Chinese translations.

Hi I saw your mod on steam and I quite like it, so I made some translations. It's not completed nor perfect but I plan to fix that in the future.
Nice mod!

woutkolkman commented:
Nice job! I'll merge it when you're ready.

Could you change the folder name from 'text_zh-cn' to 'text_chi'? That way the game will properly detect and activate these translations.
(All vanilla language folder names are: 'text_chi', 'text_eng', 'text_fre', 'text_ger', 'text_ita', 'text_jap', 'text_kor', 'text_por', 'text_rus', 'text_spa')

If it helps speed up the process, I've attached a file with the same translations, but without the '|' part. That way it can be sent to DeepL or Google Translate to rapidly translate all lines. Make sure to keep the \<LINE\> parts in the translated texts.
[BaseLanguage.txt](https://github.com/user-attachments/files/20140820/BaseLanguage.txt)

heerxingen committed:
Changed folder name to text_chi

For some reason Rain World never made Traditional Chinese translations.. I guess calling Simplified   Chinese 'chi' is a bit strange but still make some sense.

heerxingen commented:
Rain World never made offical Traditional Chinese translations so I guess it makes some sense calling Simplified Chinese 'chi'.... Though it seems a bit funny XD
And thanks for the tip on <LINE>, I was wondering what that meant

heerxingen commented:
@woutkolkman btw I was asking for advice on a QQ group of Rain World players and someone metioned that maybe "delete" would be better than "remove" in actions like "Destroy dead creatures in room", because it sounds better

*Just for the reference her qq ID is 3922319131

heerxingen committed:
Done some more translations

woutkolkman commented:
The \<LINE\> parts are used to put text on a new line or row. This way large texts fit on the screen. Rain World itself uses these brackets \<\> for special things in translations. You can find more examples here: \Steam\SteamApps\common\Rain World\RainWorld_Data\StreamingAssets\text\text_chi\strings.txt

According to the Rain World [store page](https://store.steampowered.com/app/312520/_/#:~:text=%E5%AD%97%E5%B9%95-,%E7%AE%80%E4%BD%93%E4%B8%AD%E6%96%87,-%E2%9C%94), Rain World uses Simplified Chinese translations. I don't know if this is true, but I guess the use of the 'text_chi' folder is correct for a mod.

The 'Destroy' in the texts was done intentionally, because it is used in the game's code to remove PhysicalObjects. Like [this](https://github.com/woutkolkman/mousedrag/blob/965b61e9316ff488f81ee80663b3a7907b39b57e/MouseDrag/Tools/Destroy.cs#L28).
If it makes sense in the translations, you may definitely translate it to remove or delete in Chinese.

Let me know if you need help. I'd imagine it is cumbersome with so much text. I can maybe pre-fill lines using DeepL, but I cannot read nor verify the translation unfortunately.

heerxingen commented:
yes, Rain World _does_ only have Simplified Chinese translations (In game it's just labled 'Chinese'), so using 'chi' is fine

I see what you mean by Destroy now, I'll just translated it into 'delete' since there's the other option called 'kill' which is a bit hard to distinguish in Chinese

As you probably know, on big translation projects, each language requires translators and proofreaders, but i guess for small mods and such it's hard to find so much people interested :\

heerxingen committed:
Translated Destroy to 删除 instead of 摧毁

since it sounds better
special thanks to 雾湖

heerxingen commented:
There's a typo on line 35 of the original text
you typed 'scug' but I think u meant slug, for slugcats
right?

heerxingen committed:
Updated some translations

WHY IS THERE SO MUCH TEXT x_x

woutkolkman commented:
Yes, "scug" is a typo, but it was done intentionally because it's a joke from the Discord server. The word is a mix of 'slug' and 'cat' = 'scug'. Very funny i know xP

I fixed some bugs related to translations in the master branch. Now you should be able to test Chinese translations in-game without issues. For examples I've added all other languages with a translation tool (not proofread yet).

Thank you very much for your contributions! Please let me know when you'd like the changes to be merged.

heerxingen committed:
changed this difficult line

zzz

heerxingen committed:
update

woutkolkman commented:
@heerxingen if you're OK with that, I'll merge your current pull request and add the remaining translations with DeepL. My guess is that 90% of people don't even look at the remix options and just keep the default settings. You've already translated the most common functionality.

heerxingen commented:
Yes I think that's reasonable XD
If I have time I can still edit some of those parts in the future
I've been through a lot of school work these few weeks and hasn't got the time to do the translations and frankly speaking I think I've got most of the fun parts done
Like, I don't think any Chinese has ever seen the forcefeild option :D that seems real fun
And the gravity one



---- Replied Message ----
| From | Maxi ***@***.***> |
| Date | 06/07/2025 02:41 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [woutkolkman/mousedrag] Done parts of Simplified Chinese translations. (PR #1) |
woutkolkman left a comment (woutkolkman/mousedrag#1)

@heerxingen if you're OK with that, I'll merge your current pull request and add the remaining translations with DeepL. My guess is that 90% of people don't even look at the remix options and just keep the default settings. You've already translated the most common functionality.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>

woutkolkman committed:
filled remaining text with DeepL translations
